web/Dockerfile

```diff
@@ -10,4 +10,4 @@ ADD ./package-lock.json .
 
 RUN npm install
 
-CMD npm run start
+CMD npm run start -- --public=0.0.0.0:8080
```

Comment:
Tell webpack-dev-server what address to write into the client JS it builds, so it knows how to connect back.  By default it goes to `localhost:[webpack port]`, but since we're inside a Docker container, we need to go to that container's port instead.